src/anm/analytics.js

```diff
@@ -33,40 +43,41 @@ var Analytics = function () {
     this.trackPlayingComplete = this.trackPlayer('playing_complete');
 };
 
-Analytics.prototype.track = function track(name, opts) {
-    opts = opts || {};
-    opts.name = name;
-    opts.referer = document.referrer;
-    opts.lang = navigator.language || navigator.userLanguage;
-    opts.url = location.href;
-    opts.screenHeight = screen.height;
-    opts.screenWidth = screen.width;
-    opts.windowHeight = window.innerHeight;
-    opts.windowWidth = window.innerWidth;
-    opts.timestamp = new Date().getTime();
+Analytics.prototype.track = function track(name, player, action) {
     if (this.enabled) {
-        this.queue.push(opts);
+        this.queue[player.viewId] = this.queue[player.viewId] || {
+                viewId: player.viewId,
+                projectId: player.anim.meta._anm_id,
```

Comment:
> seriously? How come?

> generated on player instance creation

you answered this yourself. player may stop one project and and play another inside one instance.